cdk/domino_cdk/agent.py

```diff
@@ -27,6 +27,11 @@ def generate_install_config(
                 }
             },
         },
+        "namespaces": {
+            "platform": {"name": f"{name}-platform"},
+            "compute": {"name": f"{name}-compute"},
+            "system": {"name": f"{name}-system"},
```

Comment:
I ended up omitting these intentionally and just going with the default `domino-xxx`. Seemed like in the non-deployer world nobody ever cared to customize it, and we're really only scripting (and setup to do) single-Domino installs anyway, so I figured why not just do the simple, uniform thing?

Reply:
:+1: will revert